tests.yml

```diff
@@ -0,0 +1,66 @@
+# Run python-versions on {{ taskotron_item }} NVR build
+
+- hosts: localhost
+  remote_user: root
+  vars:
+    testcase: dist.python-versions
+    taskotron_generic_task: true
+    # below are fallback vars for local testing
+    artifacts: ./artifacts
+    taskotron_item: python-gear-0.11.0-1.fc27  # you should really override at least this :)
+    taskotron_supported_binary_arches:
+      - x86_64
+  tasks:
+    - name: Install required packages
+      dnf:
+        name: "{{ item }}"
+        state: latest
+      with_items:
```

Comment:
What Fedora version is this running on in production?

Reply:
The approach hasn't changed, we pick the image to run on based on the item tested. So in this case we parse NVR and use disttag to decide which image to spin up.